lib/assignment/range.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+const debug = require('debug')('kafka-node:Range');
```

Comment:
I think it's confusing to use virtually no es6 features except const just because node implemented const in old versions. Can you be more explicit about which versions you support in the readme?